content/composer/blueprint-creation.md

```diff
@@ -6,61 +6,67 @@ draft: false
 weight: 400
 
 ---
+# Blueprint Editing Concepts  
 
-# Stencil Management
-The composer is delivered with a list of stencils, describing all available types of Cloudify TOSCA
-nodes. In order to add new type representation the new stencil option can be used.
-
-Users can choose the add stencil button in order to fetch a types.yaml location by pointing to
-URL. The types will be added as a separate panel with a caption of the types namespace. Each
-type appears with the derived icon (from the parent type). 
+### Using and Managing Stencils 
+The composer is delivered with a list of stencils (node types), describing all available types of Cloudify TOSCA
+nodes.
+ 
+In order to add new type representation the new stencil option can be used.
+The new types will be added as a separate panel with a caption of the types namespace. 
+Each type will appear with the derived icon (from the parent type). 
 
 When a user logs in to the system he will see the list of built in stencils (types), as well as the
 stencils the user added in previous sessions.
 
-Stencils currently cannot be deleted – this functionality will be available in future releases.
+### Adding Nodes to The Blueprint
+Nodes are defined based on built in types, as well as added inline types. 
+Note that nodes are currently defined per cloud provider (i.e. OpenStack, CloudStack, etc), or as the basic Cloudify type nodes.
+When dragging nodes into the canvas areas, the composer will only allow performing legal actions. 
```

Comment:
not sure "performing legal actions" is the right term here, sounds a bit like a translation from Hebrew. 
maybe change to:
"When dragging nodes into the canvas areas, the composer allows only valid configurations. In other words, it prevents users from creating configurations that are not consisting with topology concepts, for example ..."